policy/modules/apps/screen.te

```diff
@@ -97,6 +97,8 @@ userdom_create_user_pty(screen_domain)
 userdom_setattr_user_ptys(screen_domain)
 userdom_setattr_user_ttys(screen_domain)
 
+xdg_read_config_files(screen_domain)
```

Comment:
Is tmux reading other files under $HOME/.config or is this just for the directory access?

Reply:
Also, please wrap this in a optional block so xdg module can be disabled and screen still enabled.

Reply:
@pebenito Indeed for directory access, without it I hit an error.

@dsugar100 Done.

Reply:
@pebenito Just tried it with search_dirs() and that works fine.